gulpfile.js

```diff
@@ -8,7 +8,7 @@ var insert = require('gulp-insert');
 var replace = require('gulp-replace');
 var size = require('gulp-size');
 var streamify = require('gulp-streamify');
-var uglify = require('gulp-uglify');
+var uglify = require('gulp-uglifyes');
```

Comment:
I don't like the idea of replacing an existing package just to you can use a certain syntax.
you cannot be sure whether you broke something else with this change.